neurobooth_os/tasks/test_timing/saccades_sounds.py

```diff
@@ -30,16 +20,28 @@ def countdown(period):
 
 
 class Saccade_synch(Task_Eyetracker):
-    def __init__(self, wait_center=1, target_size=0.7, num_iterations=10, monochrome=True, **kwargs):
+    def __init__(
```

Comment:
should these defaults be in a config file?

Reply:
Will move them over!

Reply:
Got around to this if you want to take another look. Config Changes: https://github.com/neurobooth/configs/pull/10